models/Performance.js

```diff
@@ -83,97 +70,78 @@ module.exports = class Performance {
 
       const query = client.query(queryString);
 
-      query.on('row', (performance) => {
-        performances.push(this.format(performance, formatString));
+      query.on('row', ({ name, openat, closeat, performdate }) => {
+        performances.push(new Performance(name, openat, closeat, performdate));
       });
+
       query.on('end', () => {
         client.end();
         resolve(performances);
       });
+
       query.on('error', (err) => {
         client.end();
         reject(err);
       });
     });
   }
 
-  findCurrent() {
+  static findNextOrOpenWindow() {
     return new Promise((resolve, reject) => {
       const client = utils.db.createClient();
-      const queryString = 'SELECT * FROM performances WHERE current LIMIT 1';
+      const queryString = 'SELECT * FROM performances WHERE now() < closeAt ORDER BY openAt ASC LIMIT 1';
 
       client.connect();
       client.on('error', (err) => reject(err));
 
       const query = client.query(queryString);
 
-      query.on('row', (result) => {
+      query.on('row', ({ id, name, openat, closeat, performdate }) => {
         client.end();
```

Comment:
I'm going to want to use this in all of the model classes in the same PR. I don't want the scope of this one to expand to more classes. The plan is to make this a part of #166 in some capacity